internal/docker/podman.go

```diff
@@ -0,0 +1,171 @@
+package docker
+
+import (
+	"context"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"sync"
+	"time"
+
+	"github.com/anchore/stereoscope/internal/log"
+	"github.com/docker/docker/client"
+	"github.com/docker/docker/pkg/homedir"
+	"github.com/pkg/errors"
+	"golang.org/x/crypto/ssh"
+)
+
+var (
+	makePodmanClientOnce sync.Once
+	defaultHost          = "ssh://core@localhost:63753/run/user/1000/podman/podman.sock?secure=false"
```

Comment:
this has what appears to be an ephemeral port (which will change frequently)

Reply:
I changed it so the full address comes from Podman's config file